Tests/Runner/DataLoaderTest.php

```diff
@@ -1560,6 +1560,86 @@ public function testWorkspaceCleanupFailure(): void
         $componentsApi->deleteConfiguration('keboola.runner-workspace-test', $configId);
     }
 
+    public function testExternallyManagedWorkspaceSuccess(): void
+    {
+        $componentId = 'keboola.runner-workspace-test';
+        $component = new Component([
+            'id' => $componentId,
+            'data' => [
+                'definition' => [
+                    'type' => 'aws-ecr',
+                    // phpcs:ignore Generic.Files.LineLength.MaxExceeded
+                    'uri' => '147946154733.dkr.ecr.us-east-1.amazonaws.com/developer-portal-v2/keboola.runner-workspace-test',
+                    'tag' => '1.6.2',
+                ],
+                'staging-storage' => [
+                    'input' => 'workspace-snowflake',
+                    'output' => 'workspace-snowflake',
+                ],
+            ],
+        ]);
+        $clientMock = $this->createMock(BranchAwareClient::class);
+        $clientMock->method('verifyToken')->willReturn($this->clientWrapper->getBasicClient()->verifyToken());
```

Comment:
Je to tu takovy zamotany s tema klientama a mockama. Tohle bych mozna presunul o par radku niz k tomu, jak se setupuji dalsi expectations na `clientMock`

Reply:
fixed